README.md

```diff
@@ -68,6 +68,7 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade Stealth (Early 2020)              |  1532:0252  |
 | Razer Blade 15 Advanced (2020)                |  1532:0253  |
 | Razer Blade 15 (Early 2020) Base              |  1532:0255  |
+| Razer Blade 15 Advanced (Mid 2021)            |  1532:0276  |
```

Comment:
Please put the device after `Razer Blade 15 Advanced (Early 2021)` so it's sorted by usb pid